### ISSUE TITLE
EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1243]
## 一言でいうと

精度/実行速度のトレードオフを上手くチューニングする手法と、チューニング元のネットワーク(EfficientNet)の提案(これはAutoMLで構築)。ネットワークの幅・深さ・入力画像の解像度(H x M)が調整対象で、これらの相関関係から係数を算出することで単一パラメーターによる同時調整を可能にしている

![image](https://user-images.githubusercontent.com/544269/58674355-66818300-838a-11e9-9e48-bd53b57c06d9.png)

### 論文リンク

https://arxiv.org/abs/1905.11946

### 著者/所属機関

Mingxing Tan, Quoc V. Le

* Google Research

### 投稿日付(yyyy/MM/dd)

2019/5/28

## コメント

紹介ブログ
https://ai.googleblog.com/2019/05/efficientnet-improving-accuracy-and.html

**Comment by icoxfog417** [comment]
実装
https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet
